### PR TITLE
refresh abuse modal (bug 1114795)

### PR DESCRIPTION
**Pull request by ngokevin**
todo: tests

flag review abuse reports to do separately

screen shot 2014-12-22 at 2 59 22 pm
screen shot 2014-12-22 at 2 59 29 pm

Depends on https://github.com/mozilla/marketplace-elements/pull/4

### CODE REVIEW

File: src/media/js/views/app.js [hunk 1]
```diff
@@ -1,13 +1,14 @@
...
-    function(caps, iarc, l10n, log, settings, tracking, utils, z) {
+    ['content-ratings', 'l10n', 'log', 'settings', 'overflow', 'tracking',
+     'utils', 'z'],
+    function(iarc, l10n, log, settings, overflow, tracking,
+             utils, z) {
     'use strict';
-
     var gettext = l10n.gettext;
-    var console = log('app');
+    var logger = log('app');
```
Comment (spasovski) [inline src/media/js/views/app.js]: Yea we should start doing this.

File: src/media/js/views/app/abuse.js [hunk 2]
```diff
@@ -1,13 +1,13 @@
...
+     'urls', 'utils', 'z'],
+    function(caps, forms, l10n, notification, nunjucks, requests, urls,
+             utils, z) {
     var gettext = l10n.gettext;
     var notify = notification.notification;
 
-    z.page.on('submit', '.abuse-form', function(e) {
-        e.preventDefault();
-        // Submit report abuse form
+    z.doc.on('submit', '.abuse-form', utils._pd(function(e) {
```
Comment (spasovski) [inline src/media/js/views/app/abuse.js]: Why?

**Comment by spasovski** [comment]
r+

File: src/media/js/views/app/abuse.js (hunk 2 above)
Comment (ngokevin) [inline src/media/js/views/app/abuse.js]: the modal element lives in within body now since it isn't part of the main page.

**Pull request merged**